images/tags.md

```diff
@@ -32,7 +34,7 @@ that image.
 
 ![Set Default Tag](../assets/default-tag.png)
 
-To use an existing tag as the default tag, click the 3 dot menu for a tag and
-select **Make default**.
+To use an existing tag as the default tag, click the three-dot menu for a tag
```

Comment:
The downside of these kitchen-sink overflow menu buttons is that they are not super easy to describe. These are sometimes called the Pancake or [Hamburger button](https://en.wikipedia.org/wiki/Hamburger_button). I guess in this case it's an ellipsis button?

![image](https://user-images.githubusercontent.com/52710/108448318-57a1ca80-7216-11eb-9680-a04dd88a86cd.png)

https://twitter.com/lukew/status/591296890030915585

Reply:
I think I've used vertical ellipses elsewhere, so I think I'll do so here? I think kebab menu might be confusing (or am I the only person who hasn't heard it called this?)

Reply:
Oh, I think Luke was being a bit tongue-in-cheek with the names. Yeah vertical ellipsis makes sense. I was just making a general remark about these buttons being difficult to describe because they're a relatively recent (on the human scale of things) invention.